### PR TITLE
refactor: allow fallthrough when definition imlementation is not found

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/11" target="_blank">Graphite</a> 
    * **PR #12** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/12" target="_blank">Graphite</a> 
      * **PR #14** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/14" target="_blank">Graphite</a> 
        * **PR #13** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/13" target="_blank">Graphite</a> 
          * **PR #15** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/15" target="_blank">Graphite</a> 
            * **PR #16** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/16" target="_blank">Graphite</a> 
              * **PR #17** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/17" target="_blank">Graphite</a> 
                * **PR #18** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/18" target="_blank">Graphite</a> 
                  * **PR #19** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/19" target="_blank">Graphite</a> 
                    * **PR #20** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/20" target="_blank">Graphite</a> 
                      * **PR #21** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/21" target="_blank">Graphite</a> 
                        * **PR #22** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/22" target="_blank">Graphite</a> 
                          * **PR #23** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/23" target="_blank">Graphite</a> 
                            * **PR #24** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/24" target="_blank">Graphite</a> 
                              * **PR #25** <a href="https://app.graphite.dev/github/pr/skyleague/event-horizon/25" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/25?utm_source=stack-comment).

File: src/lib/event.ts [hunk 1]
```diff
@@ -40,15 +40,17 @@ export type EventHandler<C = unknown, S = unknown> =
 
 export async function handleEvent(definition: EventHandler, request: RawRequest, context: LambdaContext) {
     if ('headers' in request) {
-        return handleHttpEvent(definition, request, context)
+        if ('http' in definition) {
```
Comment (MickVanDuijn) [inline src/lib/event.ts:43]: if we check for `definition.http !== undefined`, we can skip the `notImplemented in the `handleHttpEvent`, right?

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/25?utm_source=stack-comment).


Job Reference: 74QS0D26BjCoc1fjew09

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/event-horizon/25?utm_source=stack-comment).


Job Reference: ZixEesAXj0mMWpqHV7EA

**Pull request merged**